wallet/commands.go

```diff
@@ -284,8 +284,8 @@ func (wal *Wallet) GetTransaction(walletID int, txnHash string) {
 			wal.Send <- resp
 			return
 		}
-
-		txn, err := wall.GetTransactionRaw(hash[:])
+		txn, err := wall.GetTransactionRaw(hash.String())
+		log.Info(hash.String(), txn.Amount)
```

Comment:
This change here is not part of the task are working on is it?

Reply:
This PR have to update to latest version of dcrlibwallet, some function have to change to compatible with upstream